src/Stratis.Bitcoin/Consensus/BestChainIndexer.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Consensus
+{
+    /// <summary>
+    /// An indexer of the consensus chain.
+    /// </summary>
+    public class BestChainIndexer
+    {
+        private readonly IConsensusManager consensusManager;
```

Comment:
Please add empty lines between variables.

Reply:
Why? Thats not a policy rule is it?

Reply:
Not sure but we do that everywhere so for the sake of consistency it makes sense

Reply:
Well no I tend to keep such interfaces without spaces, its not consistency because some parts of the code are set that way

Reply:
Okay